rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1032,10 +1032,55 @@ private void processReleaseRequests() {
                 activations
         );
 
+        // We have a BTC transaction, mark the UTXOs as spent and add the tx to the release set.
+        List<UTXO> availableUTXOs;
+        ReleaseTransactionSet releaseTransactionSet;
+        // Attempt access to storage first
+        // (any of these could fail and would invalidate both the tx build and utxo selection, so treat as atomic)
+        try {
+            availableUTXOs = getActiveFederationBtcUTXOs();
+            releaseTransactionSet = provider.getReleaseTransactionSet();
+        } catch (IOException exception) {
+            // Unexpected error accessing storage, log and fail
+            logger.error("Unexpected error accessing storage while attempting to processReleaseRequests ", exception);
+            return;
+        }
+
+        if (activations.isActive(RSKIP271)) {
+            processReleasesInBatch(releaseRequestQueue, txBuilder, availableUTXOs, releaseTransactionSet, activeFederationWallet);
+        } else {
+            processReleasesIndividually(releaseRequestQueue, txBuilder, availableUTXOs, releaseTransactionSet, activeFederationWallet);
+        }
+    }
+
+    private void addPegoutTxToReleaseTransactionSet(BtcTransaction generatedTransaction,
+                                                 ReleaseTransactionSet releaseTransactionSet,
+                                                 ReleaseRequestQueue.Entry releaseRequest) {
```

Comment:
We follow a slightly different convention for indentation.
```suggestion
    private void addPegoutTxToReleaseTransactionSet(
       BtcTransaction generatedTransaction,
       ReleaseTransactionSet releaseTransactionSet,
       ReleaseRequestQueue.Entry releaseRequest
    ) {
```

Reply:
Resolved